pkg/istio/rbac/common/util.go

```diff
@@ -246,25 +287,31 @@ func ParseAssertionResource(domainName zms.DomainName, assertion *zms.Assertion)
 
 // CheckAthenzSystemDisabled checks if athenz domain is systematically disabled, if so, controller skips processing current
 // role member
-func CheckAthenzSystemDisabled(roleMember *zms.RoleMember) (bool, error) {
-	if roleMember == nil {
-		return false, fmt.Errorf("got an empty role Member: %s, skipping", roleMember.MemberName)
+func CheckAthenzSystemDisabled(member interface{}) (bool, error) {
+	if member == nil {
+		return false, fmt.Errorf("got an empty role Member: %s, skipping", GetMemberName(member))
```

Comment:
Should we remove the `%s` from this string and also the GetMemberName call? Since it's nil, it will only return the empty string